neo4j-musicians/src/main/java/micronaut/neo4j/example/configuration/AppConfiguration.java

```diff
@@ -0,0 +1,21 @@
+package micronaut.neo4j.example.configuration;
+
+import io.micronaut.context.annotation.Bean;
+import io.micronaut.context.annotation.Factory;
+import org.neo4j.ogm.config.Configuration;
+import org.neo4j.ogm.session.SessionFactory;
+
+@Factory
+public class AppConfiguration {
+
+    @Bean
```

Comment:
`@Bean` shouldn't be used by itself. Use `@Prototype` if this should really be a prototype bean, or `@Singleton` if it should be a singleton

Reply:
Fixed!

Reply:
@jameskleeh hi! any news on this? :-)